tests/output/pjs/async_test.js

```diff
@@ -218,15 +218,19 @@ describe("LoopProtector", function() {
             };
         });
 
+        var code2 = getCodeFromOptions(function() {
+            mouseClicked();
```

Comment:
There isn't really an advantage to using `getCodeFromOptions` here, but it doesn't hurt any either either.